src/pages/Details.jsx

```diff
@@ -0,0 +1,65 @@
+import { faCartShopping } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { useContext } from "react";
+import { useParams } from "react-router-dom";
+import QuantityCounter from "../components/Quantity/QuantityCounter";
+import { ItemsContext } from "../context/ItemsProvider";
+import useQuantity from "../hooks/useQuantity";
+
+const Details = () => {
+  const { itemId } = useParams();
+  const { isLoading } = useContext(ItemsContext);
```

Comment:
Here you can get the `items` from the context instead of using localStorage

```suggestion
  const { items, isLoading } = useContext(ItemsContext);
```